lib/mayaUsd/render/vp2RenderDelegate/material.h

```diff
@@ -123,7 +136,11 @@ class HdVP2Material final : public HdMaterial
     SdfPath              _surfaceShaderId;  //!< Path of the surface shader
     HdVP2TextureMap      _textureMap;       //!< Textures used by this material
     TfTokenVector        _requiredPrimvars; //!< primvars required by this material
-
+#ifdef WANT_MATERIALX_BUILD
+    MaterialX::FileSearchPath _mtlxSearchPath; //!< MaterialX library search path
```

Comment:
I'm on the fence here...some part of me would like to hide these details inside of a cpp file and not expose them in the header file.

Reply:
Agreed.